microcosm_logging/factories.py

```diff
@@ -113,21 +114,22 @@ def make_json_formatter(graph):
     Used for searchable aggregation.
 
     """
-    fields = {
-        # what was logged?
-        "loggerName": "%(name)s",
-        "levelName": "%(levelname)s",
-        "message": "%(message)s",
-        # when was it logged?
-        "ascTime": "%(asctime)s",
-        # where in the code was it logged?
-        "fileName": "%(filename)s",
-        "functionName": "%(funcName)s",
-        "lineNo": "%(lineno)d",
-    }
+    fields = [
+        'asctime',
+        'levelname',
+        'name',
+        'filename',
+        'funcName',
+        'lineno',
+        'message',
+    ]
+
+    log_format = lambda x: ['%({0:s})'.format(i) for i in x]
```

Comment:
There are two arguments to the return statement. This form assigns both arguments instead of having one as a variable and other as a string constant.